src/ds5/ds5-options.cpp

```diff
@@ -655,20 +655,25 @@ namespace librealsense
             return _uvc_option->is_enabled();
     }
 
-    auto_exposure_limit_option::auto_exposure_limit_option(hw_monitor& hwm, sensor_base* ep, option_range range)
-        : option_base(range), _hwm(hwm), _sensor(ep)
+    auto_exposure_limit_option::auto_exposure_limit_option(hw_monitor& hwm, sensor_base* ep, option_range range, limits_option* exposure_limit_enable)
+        : option_base(range), _hwm(hwm), _sensor(ep), _exposure_limit_enable(exposure_limit_enable)
     {
         _range = [range]()
         {
             return range;
         };
+        set(get_range().max);
```

Comment:
Why? We don't want to modify FW's state unless requested by user

Reply:
We agreed that both controls limit values are initially set to max value